config/locales/views.fr.yml

```
@@ -253,6 +253,7 @@ fr:
       description:
         default_example: Je cherche à faire un site web pour mon magasin, j’ai besoin de conseils.
         label: Description
+        partner_url: Url de la page où se situe l'iframe
```

```suggestion
        partner_url: URL de la page où se situe l’iframe
```
(apostrophe et majuscules pour les acronymes)